articles/service-fabric/service-fabric-containers-volume-logging-drivers.md

```diff
@@ -164,11 +165,15 @@ In the **Volume** element in the snippet above, the Azure Files volume plugin re
 
 As shown in the **DriverOption** elements in the snippet above, the Azure Files volume plugin supports the following driver options:
 
+Required driver options:
 - **shareName** - Name of the Azure Files file share that provides the volume for the container
 - **storageAccountName** - Name of the Azure storage account that contains the Azure Files file share
 - **storageAccountKey** - Access key for the Azure storage account that contains the Azure Files file share
 
-All of the above driver options are required.
+Optional driver options:
+- **storageAccountFQDN** - Domain Name of the Azure Files file share. If storageAccountFQDN is not in the options, domain name will be formed by using the default suffix(.file.core.windows.net) with the storageAccountName. In the following examples, aftest is storageAccountName for general Azure and aftest2 is storageAccountName for Azure China.

```

Comment:
Provide examples of both scenarios - the one that uses the default suffix and one that uses custom suffix.